tools/topology/topology2/production/tplg-targets-ace2.cmake

```diff
@@ -21,4 +21,18 @@ SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 # No SDW Jack. SDW DMIC+SPK
 "cavs-sdw\;sof-lnl-rt1318-l12-rt714-l0\;PLATFORM=lnl,SDW_JACK=false,SDW_DMIC=1,\
 NUM_SDW_AMP_LINKS=2,SDW_DMIC_STREAM=SDW0-Capture"
+
+# SDW Bridge to SPK
+"cavs-sdw\;sof-lnl-cs42l43-l0\;PLATFORM=lnl,SDW_DMIC=1,NUM_SDW_AMP_LINKS=1,\
```

Comment:
Minor nit but its a bit weird SDW_DMIC and NUM_SDW_AMP_LINKS is in a different order on this entry.